cmd/earthly/main.go

```diff
@@ -1065,8 +1065,7 @@ func (app *earthlyApp) insertZSHCompleteEntry() error {
 func (app *earthlyApp) run(ctx context.Context, args []string) int {
 	err := app.cliApp.RunContext(ctx, args)
 
-	rpcRegex := regexp.MustCompile(`rpc error: code = .+ desc = .+:\s`)
-
+	rpcRegex := regexp.MustCompile(`(?U)rpc error: code = .+ desc = .+:\s`)
```

Comment:
Ancillary fix: Root errors with `:` were getting trimmed. This makes the regex ungreedy, so it will only prune the `rpc error` portion.